submissions/Ant-C-tech/dom-api-task/js/app.js

```diff
@@ -0,0 +1,157 @@
+    }
+}
+
+function recalcContentBlock() {
+    contentSection.style.minHeight = mapContainer.offsetHeight + 'px'
+}
+
+function showContent(param) {
+    const content = createContent(param)
+    contentSection.appendChild(content)
+    contentSection.classList.remove('content-hide')
+}
+
+function hideContent() {
+    contentSection.classList.add('content-hide')
+}
+
+function changeContent(event) {
+    //Remove all clickListeners for animation time - solution for problem with fast change content
+
+    if (event.target.classList.contains('land-active')) {
+        return false
+    } else {
+        //Map visualization
+        for (const land of mapAreas) {
+            land.classList.remove('land-active')
+        }
+        map.removeEventListener('click', changeContent)
+        event.target.classList.toggle('land-active')
+
+        //Change content
+        country.removeEventListener('click', changeContent)
+        hideContent()
+        contentSection.addEventListener('transitionend', function () {
+            contentSection.innerHTML = ''
+            showContent(content[event.target.id])
+            contentSection.addEventListener('transitionend', function () {
+                map.addEventListener('click', changeContent)
+                country.addEventListener('click', changeContent)
+            }, { once: true })
+        }, { once: true })
+    }
+}
+
+function createContent(content) {
+    let fragment = new DocumentFragment()
+
+    let titleBlock = document.createElement('div')
+    let title = document.createElement('div')
+    let subtitle = document.createElement('div')
+
+    titleBlock.classList.add('content__titleBlock')
+    title.classList.add('content__title')
+    subtitle.classList.add('content__subtitle')
+
+    title.textContent = content.title
+    subtitle.textContent = content.subtitle
+
+    titleBlock.appendChild(title)
+    titleBlock.appendChild(subtitle)
+    fragment.appendChild(titleBlock)
+
+    let textBlock = document.createElement('div')
+    textBlock.classList.add('content__textBlock')
+    for (let item of content.text) {
+        let p = document.createElement('p')
+        p.textContent = item
+        textBlock.appendChild(p)
+    }
+    fragment.appendChild(textBlock)
+
+    let imgBlock = document.createElement('div')
+    imgBlock.classList.add('content__imgBlock')
+    for (let item of content.media) {
+        let figure = document.createElement('figure')
+        figure.classList.add('imgBlock__item')
+        let img = document.createElement('img')
+        img.setAttribute('src', `${item.src}`)
+        img.setAttribute('alt', `${item.descript}`)
+        figure.appendChild(img)
+        let figcaption = document.createElement('figcaption')
+        figcaption.classList.add('imgBlock__caption')
+        figcaption.textContent = item.descript
+        figure.appendChild(figcaption)
+        imgBlock.appendChild(figure)
+    }
+    fragment.appendChild(imgBlock)
```

Comment:
You can append many elements with one `append`

 [append](https://developer.mozilla.org/en-US/docs/Web/API/ParentNode/append)